emscripten.py

```diff
@@ -401,7 +432,7 @@ def finalize_wasm(infile, outfile, memfile, DEBUG):
     args.append('-g')
   if shared.Settings.WASM_BIGINT:
     args.append('--bigint')
-  if shared.Settings.USE_LEGACY_DYNCALLS:
+  if True:##shared.Settings.USE_LEGACY_DYNCALLS:
     # we need to add all dyncalls to the wasm
     modify_wasm = True
```

Comment:
I guess you want to remove this entire if/else, and we can also remove the all the dynCall generation from binaryen?     Or at least that would be really cool if we could.

Reply:
If we have a user-facing option WASM_DYNCALLS, then this might be needed when that is enabled.